aws-samples/eks-workshop
# Update Istio lab for version 1.9.0

*Issue #, if available:*

*Description of changes:*
* update lab for latest istio version
* update installation and cleanup
* update instruction with new image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
